k8sutils/statefulset.go

```diff
@@ -47,11 +47,19 @@ type containerParameters struct {
 	PersistenceEnabled           *bool
 }
 
+type sidecarParameters struct {
```

Comment:
Why not reuse the Sidecar struct? Fields are identical appart from the json annotations.

Reply:
Would be a good use-case for struct composition so we do not embed annotations where we do not need them while reusing the same fields.

Reply:
I've transfert the k8sutils with the api1/v1beta one. Previously I had some issues with using the same one, but I made a 2nd check and everything worked fine this time.